File: src/main/scala/org/openapitools/server/model/SuggestPricesResponse.scala
package org.openapitools.server.model


/**
 * Ответ на запрос списка цен для продвижения.
 *
 * @param status  for example: ''null''
 * @param result  for example: ''null''
*/
final case class SuggestPricesResponse (
  status: Option[ApiResponseStatusType] = None,
  result: Option[SuggestPricesResultDTO] = None
)

File: ympa_fsharp_giraffe_server/src/model/SuggestPricesResponse.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.ApiResponseStatusType
open ympa_fsharp_giraffe_server.Model.SuggestPricesResultDTO

module SuggestPricesResponse =

  //#region SuggestPricesResponse


  type SuggestPricesResponse = {
    Status : ApiResponseStatusType;
    Result : SuggestPricesResultDTO;
  }
  //#endregion
